comsave/salesforce-outbound-message-bundle/0.3/config/comsave_outbound_message.yaml

```diff
@@ -0,0 +1,7 @@
+comsave_salesforce_outbound_message:
+    wsdl_directory: 'src/Company/Resources/wsdl'
+    document_paths:
+        Account:
+            path: 'src/Company/Document/Account'
+        Product2:
+            path: 'src/Company/Document/Product2'
```

Comment:
These classes does not exist. Does it make sense to add them in the recipe?

Reply:
That's correct, I don't think it makes a lot of sense to add them. They are very specific for each users own implementation. I just added the paths like this to give a fairly clear example of what a working configuration would look like. 

Does it make more sense to make them a more obscure example so it's more clear that they are just examples? Something like `path/to/document/Account`